site/_data/templates.yaml

```diff
@@ -699,6 +699,56 @@ For example, `method.method` will indicate the type of method, `authenticator`,
 
 [.deprecated]#Removed in version 1.26.0#
 "
+# OAuth two-factor enable
+- displayName: OAuth two-factor enable
+  fieldName: oauth2TwoFactorEnable
+  description: This page contains a form providing a user with the Oauth2 two-factor enable form
+  path: /oauth2/two-factor-enable
+  version: 1.42.0
+  variables:
+  - name: action
+    type: String
+    description: Optional.  If `send`, sends a Two Factor authentication code to assist in setting up Two Factor authentication.
```

Comment:
Not sure that we need to document this - this isn't something in the template for the end user to utilize, instead it is a value we set in the form data to tell FusionAuth what action you're trying to complete on form submit.